examples/simple-service-poc/simple_service.py

```diff
@@ -39,9 +39,10 @@ class SimpleService(Service):
     SIMPLE_SERVICE = "/golem/run/simple_service.py"
     SIMPLE_SERVICE_CTL = "/golem/run/simulate_observations_ctl.py"
 
-    def __init__(self, *args, instance_name, **kwargs):
+    def __init__(self, *args, instance_name, show_usage: bool = False, **kwargs):
```

Comment:
I'd either specify types for both `instance_name` and `show_usage`, or for neither of them.

Reply:
I mean, here, done ;)